src/cli/commit/args.rs

```diff
@@ -316,32 +307,6 @@ mod tests {
 
         Ok(())
     }
-    fn fake_project_dir() -> anyhow::Result<ProjectDirs> {
-        let dirs = ProjectDirs::from(&format!("{}", Uuid::new_v4()), "xsv24", "git-kit")
-            .expect("Failed to retrieve 'git-kit' config");
-
-        Ok(dirs)
-    }
-
-    fn clean<C: GitCommands, S: Store>(context: AppContext<C, S>) -> anyhow::Result<()> {
```

Comment:
We don't need this since we are faking the config an no files are created for tests.